diff -u tclx8.3.5/tcl/unix/Makefile.in~ tclx8.3.5/tcl/unix/Makefile.in
--- tclx8.3.5/tcl/unix/Makefile.in~	2004-01-30 18:31:32.000000000 +0900
+++ tclx8.3.5/tcl/unix/Makefile.in	2004-01-30 18:31:32.000000000 +0900
@@ -136,7 +136,7 @@
 #
 all: binaries libraries doc
 
-binaries: tcl checkup RUNTIME tclXtest
+binaries: tcl checkup RUNTIME
 
 libraries: tcl.tndx tcl.tlib tclx.tcl buildidx.tcl
 
diff -u tclx8.3.5/tk/unix/Makefile.in~ tclx8.3.5/tk/unix/Makefile.in
--- tclx8.3.5/tk/unix/Makefile.in~	2004-01-30 18:43:13.000000000 +0900
+++ tclx8.3.5/tk/unix/Makefile.in	2004-01-30 18:43:13.000000000 +0900
@@ -85,7 +85,7 @@
 #
 all: binaries libraries doc
 
-binaries: wishx tktest
+binaries: wishx
 
 libraries: RUNTIME
 
